faw/pdf-observatory/main.py

```diff
@@ -111,7 +111,7 @@ def main(pdf_dir, mongodb, host, port, hostname, in_docker, production, config,
     # https://docs.mongodb.com/manual/reference/limits/
     async def admin_cfg():
         await app_mongodb_conn.client.admin.command({
-                'setFeatureCompatibilityVersion': '4.4'})
+                'setFeatureCompatibilityVersion': '6.0'})
```

Comment:
From mongo's docs:

> Enabling backwards-incompatible features can complicate the downgrade process since you must remove any persisted backwards-incompatible features before you downgrade.
> 
> It is recommended that after upgrading, you allow your deployment to run without enabling backwards-incompatible features for a burn-in period to ensure the likelihood of downgrade is minimal. When you are confident that the likelihood of downgrade is minimal, enable these features.

Any mongo version >=5.0 does not allow setting this to anything less than 5.0.